src/components/notifications/notifications.scss

```diff
@@ -367,3 +367,31 @@
         fill: var(--Y700);
     }
 }
+
+.hover-only {
```

Comment:
wrap these in some container to avoid overlap

Reply:
done